docs/quickstarts/ratify-with-aws-signer.md

```diff
@@ -233,14 +233,14 @@ cat > notation-verifier.yaml << EOF
 apiVersion: config.ratify.deislabs.io/v1beta1
 kind: Verifier
 metadata:
-  name: verifier-notary
+  name: verifier-notation
 spec:
-  name: notaryv2
+  name: notation
   artifactTypes: application/vnd.cncf.notary.signature
```

Comment:
is this the correct artifact type now?

Reply:
It's correct artifact type, see https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md#backward-compatibility